5-infrastructure/modules/bastion/README.md

```diff
@@ -0,0 +1,27 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
nit: add a header with title

Reply:
Done in 0d07125489fd5149041f17a34f77eb7da4623c5c